JustSaying.IntegrationTests/AwsTools/BasicHandlingThrottlingTest.cs

```diff
@@ -81,7 +81,7 @@ public async Task HandlingManyMessages(int throttleMessageCount)
                     entriesAdded++;
                 }
 
-                await client.SendMessageBatchAsync(queue.Url, entries);
+                await client.SendMessageBatchAsync(queue.Uri.ToString(), entries);

```

Comment:
Use `AbsoluteUri` rather than `ToString()`? (And all the other places in the diff).